src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.scss

```diff
@@ -1,10 +1,6 @@
 @use 'sass:map';
 @use '@angular/material' as mat;
-
-@import '../../../../styles.scss';
-
-$color-config: mat.get-color-config($planscape-frontend-theme);
-$primary-palette: map.get($color-config, 'primary');
+@import "../../../../theme";
```

Comment:
nit: for consistency, should we have `.scss` here, as we do in [map-control-panel.component.scss](https://github.com/OurPlanscape/Planscape/pull/1076/files#diff-aca2dc9a47ee1372113ef49a4a4c769486234a9e686bfcbd3172affee1c1c03a)?

Reply:
By looking at https://sass-lang.com/documentation/at-rules/import/ it seems like the convention is the other way around (no .scss)